### ISSUE TITLE
Integration tests

### ISSUE THREAD

**Issue by JosephKiranBabu** [freedombox/Plinth#863]
Setup integration tests for Plinth and run them on the CI server.

@jvalleroy Are these tests still relevant? https://github.com/jvalleroy/plinth-tester
Now that we have a CI server, maybe we can start by running these tests.

**Comment by jvalleroy** [comment]
@JosephKiranBabu Yes, but I haven't updated it in some time. These are more like acceptance tests that use a browser to interact with a VM (you can use xvfb to run on a server without desktop installed). Most of them will install an app, try enabling/disabling, and check that the service is running or webpage is accessible.
